resources/managed/all-osd-resources.yaml

```diff
@@ -83,18 +83,12 @@ Resources:
     name: osd-delete-backplane-serviceaccounts
   - namespace: openshift-build-test
     name: sre-build-test
-  - namespace: openshift-cloud-ingress-operator
```

Comment:
Is it expected that this and the other service accounts are being removed as managed resources in this PR? It looks like it is removing all the SAs that are listed/deployed via `deploy/osd-serviceaccounts`.

Reply:
Interesting, looks like rerunning the script fixed the missing serviceaccounts. Thanks for catching this!